backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/session/UserSession.java

```diff
@@ -477,6 +477,12 @@ public int getHttpCacheMaxAge()
 		};
 	}
 
+	private static final String SYSCONFIG_isAlwaysShowNewBPartner = "de.metas.ui.web.session.UserSession.IsAlwaysDisplayNewBPartner";
```

Comment:
* suggestion: stick with "Display" instead of "Show"
* please add a javadoc which indicates that this is about an item / option thingie in the UI (AFAIU it is, right?)